nimare/transforms.py

```diff
@@ -475,6 +475,45 @@ def transform(self, dataset):
         return new_dataset
 
 
+class StandardizeField(NiMAREBase):
```

Comment:
should have a test for this new class

Reply:
added a test for `StandardizeField` class.